portfolio/src/main/webapp/index.html

```diff
@@ -24,8 +66,30 @@ <h1 style="background-color:SlateBlue;">Anesia's Portfolio</h1>
             Feel free to add me if you like.</p>
         <p> Also check out my <a href="https://github.com/hoil200">Github</a>
             if you so desire.</p>
-        <button onclick = "getData()">Click for a medical show</button>
-        <div id = "show-container"></div>
+        <h1> School Activities </h1>
+        <div id="chart-container"></div>
+        <body onload= "drawChart()">
+
+        </body>    
+        
+        <body>   
+            
+        <form action = "/data" method = "post">
+          <h2>Leave a comment here</h2>
+          <textarea name = "text-input"></textarea>
+          <br/><br/>
+          
+          <input type = "submit" />
+        </form>
+        <br/>
+
+        <button onclick="loadComments">Show Comments </button> 
```

Comment:
Ah.  So it looks like you are probably 1/2 way through changing from storing comments to storing shows.  There will definitely be a tension sometimes between doing small incremental changes and making sure your code is always 100% self consistent after each check-in.  You can sometimes get around this by leaving in some TODOs, like "TODO: display these as Shows instead of Comments".  For this one I would probably just go ahead and finish the switch to treating what you are saving and loading from the database as shows instead of comments.